[2025-02-18T13:03:17.517+0000] {processor.py:186} INFO - Started process (PID=724) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:17.518+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:03:17.520+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:03:17.519+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:17.531+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:03:17.529+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:03:17.533+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:17.545+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.032 seconds
[2025-02-18T13:03:48.087+0000] {processor.py:186} INFO - Started process (PID=791) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:48.090+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:03:48.092+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:03:48.091+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:48.101+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:03:48.098+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:03:48.103+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:03:48.118+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.036 seconds
[2025-02-18T13:04:19.016+0000] {processor.py:186} INFO - Started process (PID=858) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:19.016+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:04:19.018+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:04:19.018+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:19.026+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:04:19.023+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:04:19.027+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:19.039+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:04:49.541+0000] {processor.py:186} INFO - Started process (PID=925) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:49.541+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:04:49.543+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:04:49.543+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:49.551+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:04:49.549+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:04:49.553+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:04:49.564+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.028 seconds
[2025-02-18T13:05:20.116+0000] {processor.py:186} INFO - Started process (PID=992) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:20.117+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:05:20.118+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:05:20.118+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:20.127+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:05:20.124+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:05:20.129+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:20.140+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.029 seconds
[2025-02-18T13:05:50.874+0000] {processor.py:186} INFO - Started process (PID=1059) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:50.875+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:05:50.877+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:05:50.877+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:50.887+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:05:50.884+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:05:50.889+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:05:50.901+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.031 seconds
[2025-02-18T13:06:20.957+0000] {processor.py:186} INFO - Started process (PID=1126) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:20.958+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:06:20.960+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:06:20.960+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:20.969+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:06:20.967+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:06:20.971+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:20.994+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.040 seconds
[2025-02-18T13:06:51.750+0000] {processor.py:186} INFO - Started process (PID=1193) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:51.750+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:06:51.752+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:06:51.752+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:51.760+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:06:51.758+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:06:51.761+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:06:51.774+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.028 seconds
[2025-02-18T13:07:22.268+0000] {processor.py:186} INFO - Started process (PID=1260) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:22.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:07:22.270+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:07:22.270+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:22.278+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:07:22.276+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:07:22.279+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:22.291+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:07:52.556+0000] {processor.py:186} INFO - Started process (PID=1327) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:52.556+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:07:52.558+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:07:52.558+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:52.566+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:07:52.564+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:07:52.567+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:07:52.579+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:08:23.454+0000] {processor.py:186} INFO - Started process (PID=1394) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:23.455+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:08:23.456+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:08:23.456+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:23.465+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:08:23.462+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:08:23.466+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:23.477+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:08:54.031+0000] {processor.py:186} INFO - Started process (PID=1461) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:54.032+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:08:54.034+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:08:54.033+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:54.042+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:08:54.040+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:08:54.043+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:08:54.056+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.029 seconds
[2025-02-18T13:09:24.864+0000] {processor.py:186} INFO - Started process (PID=1529) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:24.865+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:09:24.867+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:09:24.866+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:24.876+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:09:24.874+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:09:24.878+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:24.891+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.031 seconds
[2025-02-18T13:09:55.535+0000] {processor.py:186} INFO - Started process (PID=1596) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:55.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:09:55.537+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:09:55.537+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:55.544+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:09:55.542+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:09:55.545+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:09:55.558+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:10:26.194+0000] {processor.py:186} INFO - Started process (PID=1663) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:26.195+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:10:26.196+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:10:26.196+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:26.208+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:10:26.205+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:10:26.210+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:26.226+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.038 seconds
[2025-02-18T13:10:56.901+0000] {processor.py:186} INFO - Started process (PID=1730) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:56.902+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:10:56.903+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:10:56.902+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:56.910+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:10:56.908+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:10:56.911+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:10:56.939+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.042 seconds
[2025-02-18T13:11:27.623+0000] {processor.py:186} INFO - Started process (PID=1797) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:27.624+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:11:27.625+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:11:27.624+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:27.632+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:11:27.630+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:11:27.634+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:27.645+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.025 seconds
[2025-02-18T13:11:57.913+0000] {processor.py:186} INFO - Started process (PID=1864) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:57.914+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:11:57.915+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:11:57.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:57.922+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:11:57.920+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:11:57.924+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:11:57.935+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.026 seconds
[2025-02-18T13:12:28.226+0000] {processor.py:186} INFO - Started process (PID=1931) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:28.227+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:12:28.228+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:12:28.228+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:28.236+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:12:28.233+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:12:28.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:28.248+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.025 seconds
[2025-02-18T13:12:58.632+0000] {processor.py:186} INFO - Started process (PID=1998) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:58.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:12:58.634+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:12:58.633+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:58.641+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:12:58.639+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:12:58.643+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:12:58.655+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:13:28.912+0000] {processor.py:186} INFO - Started process (PID=2066) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:28.913+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:13:28.914+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:13:28.914+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:28.923+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:13:28.920+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:13:28.925+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:28.937+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.030 seconds
[2025-02-18T13:13:59.404+0000] {processor.py:186} INFO - Started process (PID=2133) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:59.405+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:13:59.406+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:13:59.406+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:59.413+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:13:59.411+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:13:59.415+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:13:59.426+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.026 seconds
[2025-02-18T13:14:29.716+0000] {processor.py:186} INFO - Started process (PID=2200) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:14:29.716+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:14:29.717+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:14:29.717+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:14:29.725+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:14:29.723+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:14:29.727+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:14:29.738+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.026 seconds
[2025-02-18T13:15:00.132+0000] {processor.py:186} INFO - Started process (PID=2267) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:00.132+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:15:00.133+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:15:00.133+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:00.142+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:15:00.139+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:15:00.143+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:00.155+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:15:30.481+0000] {processor.py:186} INFO - Started process (PID=2334) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:30.482+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:15:30.483+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:15:30.483+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:30.492+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:15:30.489+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:15:30.493+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:15:30.504+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:16:00.849+0000] {processor.py:186} INFO - Started process (PID=2401) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:00.850+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:16:00.850+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:16:00.850+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:00.859+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:16:00.856+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:16:00.860+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:00.871+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.026 seconds
[2025-02-18T13:16:31.293+0000] {processor.py:186} INFO - Started process (PID=2468) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:31.294+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:16:31.295+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:16:31.294+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:31.303+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:16:31.300+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:16:31.304+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:16:31.315+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.026 seconds
[2025-02-18T13:17:01.944+0000] {processor.py:186} INFO - Started process (PID=2535) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:01.945+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:17:01.946+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:17:01.946+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:01.955+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:17:01.952+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:17:01.956+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:01.969+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.029 seconds
[2025-02-18T13:17:32.462+0000] {processor.py:186} INFO - Started process (PID=2602) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:32.464+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:17:32.474+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:17:32.473+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:32.493+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:17:32.488+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:17:32.495+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:17:32.530+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.074 seconds
[2025-02-18T13:18:48.664+0000] {processor.py:186} INFO - Started process (PID=51) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:18:48.665+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:18:48.667+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:18:48.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:18:48.678+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:18:48.674+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:18:48.680+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:18:48.708+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.049 seconds
[2025-02-18T13:19:19.511+0000] {processor.py:186} INFO - Started process (PID=115) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:19.512+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:19:19.514+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:19:19.513+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:19.522+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:19:19.520+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:19:19.523+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:19.536+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.028 seconds
[2025-02-18T13:19:50.276+0000] {processor.py:186} INFO - Started process (PID=179) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:50.276+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:19:50.278+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:19:50.277+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:50.287+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:19:50.284+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:19:50.288+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:19:50.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.032 seconds
[2025-02-18T13:20:20.823+0000] {processor.py:186} INFO - Started process (PID=243) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:20.824+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:20:20.826+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:20.825+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:20.833+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:20.831+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_etl_beaty_salon.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_etl_beaty_salon.py", line 12, in <module>
    logger = logging.getLogger(name)
                               ^^^^
NameError: name 'name' is not defined
[2025-02-18T13:20:20.834+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:20.846+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.027 seconds
[2025-02-18T13:20:25.206+0000] {processor.py:186} INFO - Started process (PID=301) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:25.206+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:20:25.208+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.207+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:25.235+0000] {processor.py:925} INFO - DAG(s) 'dds_mart_transfer_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:25.298+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.297+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:dds_mart_transfer_dag
[2025-02-18T13:20:25.305+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.305+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:dds_mart_transfer_dag
[2025-02-18T13:20:25.309+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.309+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:dds_mart_transfer_dag
[2025-02-18T13:20:25.315+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.314+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:dds_mart_transfer_dag
[2025-02-18T13:20:25.319+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.319+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:dds_mart_transfer_dag
[2025-02-18T13:20:25.324+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.323+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:dds_mart_transfer_dag
[2025-02-18T13:20:25.328+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.328+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:dds_mart_transfer_dag
[2025-02-18T13:20:25.328+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.328+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:20:25.338+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.338+0000] {dag.py:3262} INFO - Creating ORM DAG for dds_mart_transfer_dag
[2025-02-18T13:20:25.346+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:25.346+0000] {dag.py:4180} INFO - Setting next_dagrun for dds_mart_transfer_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:20:25.360+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.158 seconds
[2025-02-18T13:20:55.504+0000] {processor.py:186} INFO - Started process (PID=364) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:55.505+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:20:55.506+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:55.506+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:55.531+0000] {processor.py:925} INFO - DAG(s) 'dds_mart_transfer_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:20:55.539+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:55.539+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:20:55.559+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:20:55.558+0000] {dag.py:4180} INFO - Setting next_dagrun for dds_mart_transfer_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:20:55.572+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.072 seconds
[2025-02-18T13:21:25.955+0000] {processor.py:186} INFO - Started process (PID=429) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:25.956+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:21:25.957+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:25.957+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:25.978+0000] {processor.py:925} INFO - DAG(s) 'dds_mart_transfer_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:25.985+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:25.985+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:21:26.001+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:26.000+0000] {dag.py:4180} INFO - Setting next_dagrun for dds_mart_transfer_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:21:26.011+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.060 seconds
[2025-02-18T13:21:44.562+0000] {processor.py:186} INFO - Started process (PID=437) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:44.563+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:21:44.564+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.564+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:44.600+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:21:44.668+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.668+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:ETL_dag
[2025-02-18T13:21:44.675+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.675+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:ETL_dag
[2025-02-18T13:21:44.679+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.679+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:ETL_dag
[2025-02-18T13:21:44.683+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.683+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:ETL_dag
[2025-02-18T13:21:44.686+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.686+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:ETL_dag
[2025-02-18T13:21:44.690+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.690+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:ETL_dag
[2025-02-18T13:21:44.695+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.694+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:ETL_dag
[2025-02-18T13:21:44.695+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.695+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:21:44.704+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.704+0000] {dag.py:3262} INFO - Creating ORM DAG for ETL_dag
[2025-02-18T13:21:44.711+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:21:44.711+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:21:44.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.164 seconds
[2025-02-18T13:22:14.822+0000] {processor.py:186} INFO - Started process (PID=501) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:14.823+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:22:14.825+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:14.825+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:14.849+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:14.857+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:14.856+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:22:14.872+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:14.872+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:22:14.883+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.065 seconds
[2025-02-18T13:22:34.086+0000] {processor.py:186} INFO - Started process (PID=565) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:34.087+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:22:34.088+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:34.088+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:34.118+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:22:34.191+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:34.190+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:22:34.210+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:22:34.209+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:22:34.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.147 seconds
[2025-02-18T13:23:04.390+0000] {processor.py:186} INFO - Started process (PID=629) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:04.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:23:04.392+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:04.391+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:04.413+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:04.421+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:04.421+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:23:04.437+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:04.437+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:23:04.448+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.061 seconds
[2025-02-18T13:23:34.724+0000] {processor.py:186} INFO - Started process (PID=694) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:34.725+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:23:34.726+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:34.726+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:34.751+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:23:34.760+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:34.759+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:23:34.778+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:23:34.777+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:23:34.792+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.072 seconds
[2025-02-18T13:24:04.943+0000] {processor.py:186} INFO - Started process (PID=758) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:04.944+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:24:04.945+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:04.945+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:04.970+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:04.979+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:04.979+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:24:04.997+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:04.997+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:24:05.010+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.071 seconds
[2025-02-18T13:24:35.399+0000] {processor.py:186} INFO - Started process (PID=822) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:35.400+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:24:35.401+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:35.400+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:35.422+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:24:35.429+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:35.429+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:24:35.445+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:24:35.445+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:24:35.455+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.060 seconds
[2025-02-18T13:25:05.494+0000] {processor.py:186} INFO - Started process (PID=886) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:05.495+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:25:05.495+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:05.495+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:05.518+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:05.525+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:05.525+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:25:05.541+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:05.541+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:25:05.553+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.063 seconds
[2025-02-18T13:25:36.350+0000] {processor.py:186} INFO - Started process (PID=950) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:36.350+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:25:36.351+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:36.351+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:36.374+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:25:36.382+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:36.382+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:25:36.398+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:25:36.398+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:25:36.410+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.065 seconds
[2025-02-18T13:26:06.840+0000] {processor.py:186} INFO - Started process (PID=1014) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:06.840+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:26:06.841+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:06.841+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:06.866+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:06.874+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:06.874+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:26:06.891+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:06.891+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:26:06.903+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.068 seconds
[2025-02-18T13:26:37.018+0000] {processor.py:186} INFO - Started process (PID=1077) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:37.019+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:26:37.020+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:37.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:37.045+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:26:37.054+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:37.053+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:26:37.072+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:26:37.072+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:26:37.085+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.070 seconds
[2025-02-18T13:27:07.234+0000] {processor.py:186} INFO - Started process (PID=1127) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:07.234+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:27:07.235+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:07.235+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:07.258+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:07.266+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:07.265+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:27:07.282+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:07.282+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:27:07.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.065 seconds
[2025-02-18T13:27:38.163+0000] {processor.py:186} INFO - Started process (PID=1192) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:38.164+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:27:38.165+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:38.165+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:38.189+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:27:38.198+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:38.198+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:27:38.218+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:27:38.218+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:27:38.232+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.073 seconds
[2025-02-18T13:28:08.956+0000] {processor.py:186} INFO - Started process (PID=1247) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:08.957+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:28:08.958+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:08.958+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:08.983+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:08.992+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:08.992+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:28:09.011+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:09.011+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:28:09.025+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.073 seconds
[2025-02-18T13:28:40.113+0000] {processor.py:186} INFO - Started process (PID=1311) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:40.114+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:28:40.115+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:40.114+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:40.139+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:28:40.148+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:40.148+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:28:40.170+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:28:40.170+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:28:40.191+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.082 seconds
[2025-02-18T13:30:14.933+0000] {processor.py:186} INFO - Started process (PID=52) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:14.934+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:30:14.936+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:14.936+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:14.970+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:14.991+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:14.991+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:30:15.010+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:15.010+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:30:15.023+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.096 seconds
[2025-02-18T13:30:46.003+0000] {processor.py:186} INFO - Started process (PID=117) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:46.004+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:30:46.006+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:46.005+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:46.028+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:30:46.036+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:46.036+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:30:46.053+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:30:46.052+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:30:46.064+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.066 seconds
[2025-02-18T13:31:16.523+0000] {processor.py:186} INFO - Started process (PID=182) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:16.524+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:31:16.525+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:16.525+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:16.548+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:16.556+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:16.556+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:31:16.573+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:16.573+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:31:16.584+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.065 seconds
[2025-02-18T13:31:46.856+0000] {processor.py:186} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:46.857+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:31:46.858+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:46.858+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:46.881+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:31:46.890+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:46.890+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:31:46.908+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:31:46.908+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:31:46.921+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.069 seconds
[2025-02-18T13:32:17.384+0000] {processor.py:186} INFO - Started process (PID=312) to work on /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:32:17.385+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_etl_beaty_salon.py for tasks to queue
[2025-02-18T13:32:17.387+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:32:17.386+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:32:17.409+0000] {processor.py:925} INFO - DAG(s) 'ETL_dag' retrieved from /opt/airflow/dags/dag_etl_beaty_salon.py
[2025-02-18T13:32:17.417+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:32:17.417+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-18T13:32:17.437+0000] {logging_mixin.py:190} INFO - [2025-02-18T13:32:17.437+0000] {dag.py:4180} INFO - Setting next_dagrun for ETL_dag to 2025-02-17 00:00:00+00:00, run_after=2025-02-18 00:00:00+00:00
[2025-02-18T13:32:17.450+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_etl_beaty_salon.py took 0.070 seconds
